apps/web/vitest.setup.ts

```
@@ -336,53 +165,30 @@ vi.mock('@viberater/convex', () => ({
   },
 }));
 
-// Mock @/queries module
+// Simplified mock for @/queries module
+const defaultQuery = () => ({ data: [], isLoading: false, error: null });
+const defaultMutation = () => ({ mutate: vi.fn(), isLoading: false, error: null });
```

ESLint

[error] Replace `·mutate:·vi.fn(),·isLoading:·false,·error:·null·` with `⏎··mutate:·vi.fn(),⏎··isLoading:·false,⏎··error:·null,⏎`

(prettier/prettier)